package.json

```diff
@@ -60,5 +60,9 @@
     "*.ts": [
       "npm run lint:fix"
     ]
+  },
+  "repository": {
+    "url": "github:contentful/node-apps-toolkit",
```

Comment:
TIL you can put not-URLs in the url config values: https://docs.npmjs.com/cli/v10/configuring-npm/package-json#repository